doc/en/user/source/webadmin/data/layers.rst

```diff
@@ -80,61 +86,55 @@ Clicking the layer name opens a layer configuration panel. The :guilabel:`Data`
 
   .. figure:: ../images/data_layers_meta.png
 
-     Adding a metadata link n FGDC format
+     *Adding a metadata link in FGDC format*
    
 Coordinate Reference Systems
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-A coordinate reference system (CRS) defines how your georeferenced spatial data relates to real locations on the Earth’s surface. CRSs are part of a more general model called Spatial Reference Systems (SRS), which includes referencing by coordinates and geographic identifiers. GeoServer needs to know the Coordinate Reference System of your data. This information is used for computing the latitude/longitude bounding box and reprojecting the data during both WMS and WFS requests.
+A coordinate reference system (CRS) defines how georeferenced spatial data relates to real locations on the Earth’s surface. CRSes are part of a more general model called Spatial Reference Systems (SRS), which includes referencing by coordinates and geographic identifiers. GeoServer needs to know the Coordinate Reference System of your data. This information is used for computing the latitude/longitude bounding box and reprojecting the data during both WMS and WFS requests.
 
 .. figure:: ../images/data_layers_CRS.png
    
-   *Coordinate reference system of the layer*  
+   *Coordinate reference system of a layer*  
 
-* **Native SRS**—Refers to the projection the layer is stored in. Clicking the projection link displays a description of the SRS.
-* **Declared SRS**—Refers to what GeoServer publishes to clients 
-* **SRS Handling:**—Determines how GeoServer should handle projection when the two SRS differ 
+* **Native SRS**—Specifies the coordinate system the layer is stored in. Clicking the projection link displays a description of the SRS.
+* **Declared SRS**—Specifies the coordinate system GeoServer publishes to clients 
+* **SRS Handling**—Determines how GeoServer should handle projection when the two SRSes differ 
 
 Bounding Boxes
 ^^^^^^^^^^^^^^
 
-The bounding box determines the extent of a layer. The :guilabel:`Native Bounding Box` is the bounds of the data projected in the Native SRS. You can generate these bounds by clicking the :guilabel:`Compute from data` button. The :guilabel:`Lat/Long Bounding Box` computes the bounds based on the standard lat/lon. These bounds can be generated by clicking the :guilabel:`Compute from native bounds` button.
+The bounding box determines the extent of the data within a layer. 
+
+* **Native Bounding Box**—The bounds of the data specified in the Native SRS. These bounds can be generated by clicking the :guilabel:`Compute from data` button. 
```

Comment:
Hmm... eh... native is a bit confusing here, in a sense, it's what we claim it's native to the OGC clients, not the actual native CRSs. The difference between the two is evident when one has a native CRS that cannot be matched to a EPSG code, you setup a different one as declared and this will be the the actual CRS in which this bbox should be computed... we should probably rename it to "declared bounding box". Anyways, not something you should change here, more or a clarification of something I've realized clearly just recently. See also https://osgeo-org.atlassian.net/browse/GEOS-7004